ui/app/components/clients/page/sync.ts

```diff
@@ -4,9 +4,12 @@
  */
 
 import ActivityComponent from '../activity';
-
 export default class SyncComponent extends ActivityComponent {
   title = 'Secrets sync usage';
   description =
     'This data can be used to understand how many secrets sync clients have been used for this date range. Each Vault secret that is synced to at least one destination counts as one Vault client.';
+
+  get secretsSyncActivated() {
```

Comment:
Not necessary to change as this naming is very clear, but just wanted to mention that a _soft_ pattern we have is naming getters or vars with `is` or `has` if it returns a boolean